src/client/pages/ConsentsNewsletters.stories.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable functional/immutable-data */
+import React from 'react';
+import { Meta } from '@storybook/react';
+
+import { ConsentsNewsletters } from './ConsentsNewsletters';
+
+export default {
+  title: 'Pages/ConsentsNewsletters',
+  component: ConsentsNewsletters,
+} as Meta;
+
+export const NoNewsletters = () => (
+  <ConsentsNewsletters newsletters={[]} isUserInTest={false} />
+);
+NoNewsletters.story = {
+  name: 'with no newsletters',
+};
+
+export const NoNewslettersInTest = () => (
+  <ConsentsNewsletters newsletters={[]} isUserInTest={true} />
+);
+NoNewslettersInTest.story = {
+  name: 'with no newsletters, in test',
+};
+
+export const SingleNewsletter = () => (
+  <ConsentsNewsletters
+    newsletters={[
+      {
+        id: '0',
+        nameId: 'n0',
+        description: 'Newsletter description',
+        name: 'Newsletter Name',
+      },
+    ]}
+    isUserInTest={false}
+  />
+);
+SingleNewsletter.story = {
+  name: 'with a single newsletter',
+};
+
+export const SingleNewsletterInTest = () => (
+  <ConsentsNewsletters
+    newsletters={[
+      {
+        id: '0',
+        nameId: 'n0',
+        description: 'Newsletter description',
+        name: 'Newsletter Name',
```

Comment:
again as in the previous comment, and probably more so for "in test", would be useful to get a newsletter with an image